lisp/tree-sitter-hl.el

```diff
@@ -111,12 +111,16 @@
 ;;; ------------------------------------
 ;;; Variables, properties.
 
-;;; TODO: Add variable.use?
```

Comment:
IMHO `variable.reference` would be a little more clear, so if you like that I could change it, but `.use` works too :)